filesys/fuse/BUILD

```diff
@@ -1,10 +1,4 @@
-OPTS+=" --enable-lib \
-        --enable-util \
-        --disable-static \
-        --disable-example \
-        --sbindir=/usr/bin"
-
-default_build &&
+default_meson_build &&
 
 if module_installed systemd; then
```

Comment:
We can drop the if check for systems, since that would always be present.

That init file should always be removed :)